pkg/sql/executor_normal_stmt.go

```diff
@@ -38,7 +47,7 @@ func isQuery(slct string) bool {
 	if strings.HasPrefix(s, "SELECT") && !has(s, "INTO") {
 		return true
 	}
-	if strings.HasPrefix(s, "SHOW") && (has(s, "CREATE") || has(s, "DATABASES") || has(s, "TABLES")) {
+	if strings.HasPrefix(s, "SHOW") && containsAnyString(s, "CREATE", "DATABASES", "TABLES", "COLUMNS", "TABLE", "PROCESSLIST") {
```

Comment:
For discussion, maybe we can remove this `isQuery` check and forward all standard SQL to `db.Query`? I am not sure if `db.Exec` is useful...

Reply:
`db.Exec` returns a https://golang.org/pkg/database/sql/#Result object, which seems cannot be acquired from a call to `db.Query`. This is the only difference I can found.
The [MySQL C++ client](https://dba.stackexchange.com/questions/24542/execute-vs-executequery-in-mysql-connector-c) recommends we call `execute` to check whether a statement is an INSERT, UPDATE, or DELETE statement. Unfortunately, the go-database module doesn't provide a similar mechanism.